strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -284,10 +285,26 @@ def coef(arr):
 
     def prepare_fock(self, n, r=0.0001):
```

Comment:
What would be a good r value for a given n ? 
What's the error made as a function of n and r ?